tests/src/unit/WebdamConfigFormTest.php

```diff
@@ -25,7 +26,6 @@ public function setUp() {
     \Drupal::setContainer($container);
   }
 
-
   public function testGetFormId() {
     $form = new WebdamConfig($this->getConfigFactoryStub());
```

Comment:
Yeah, mock out something that implements WebdamInterface, and then you can pass it into WebdamConfig as an argument.